cg/meta/deliver/delivery_api.py

```diff
@@ -200,10 +200,8 @@ def _is_sample_deliverable(self, sample: Sample) -> bool:
         )
 
     def _bundle_exists(self, bundle_name: str, workflow: str) -> bool:
-        if self.hk_api.last_version(bundle_name):
+        if self.hk_api.last_version(bundle_name) or self.ignore_missing_bundle(workflow):
             return True
-        if self.ignore_missing_bundle(workflow):
-            return False
         raise SyntaxError(f"Could not find any version for {bundle_name}")
```

Comment:
Should we stop raising this weird exception, log a warning and just return false instead?

Reply:
Maybe better to return False? At least I would expect a method with return type boolean to be able to return False somehow.

Reply:
Take it with a grain of salt though, I am not so well-versed in this flow.

Reply:
The logic is a mess but we need to keep marking the delivery as failed for case files.  For sample files it's already in place.